cmds/filesystem.c

```diff
@@ -479,6 +479,11 @@ static int btrfs_scan_kernel(void *search, unsigned unit_mode)
 
 		fd = open(mnt->mnt_dir, O_RDONLY);
 		if ((fd != -1) && !get_df(fd, &space_info_arg)) {
+			/* put spacing between filesystem entries for readability */
+			if (found != 0) {
+				pr_verbose(LOG_DEFAULT, "\n");
+			}
```

Comment:
For single statements it's not necessary to use { }, which is the preferred coding style.

Reply:
Got it, updated!